_data/simple-icons.json

```diff
@@ -3707,6 +3707,11 @@
             "hex": "EC1C24",
             "source": "https://facepunch.com/img/brand/default-light.svg"
         },
+        {
+            "title": "Fairphone",
+            "hex": "2FA6DF",
+            "source": "https://www.fairphone.com/"
```

Comment:
```suggestion
            "source": "https://www.fairphone.com",
            "license":  {
                "type": "CC-BY-NC-SA-4.0"
            }
```

Reply:
@adamrusted  how come the license for the logo is CC-BY-NC-SA-4.0?

Reply:
We can remove the license, as I can't recall where I saw it.